packages/dropdowns/src/styled/field/StyledMultiselect.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { StyledInput } from '../field/StyledInput';
+import { zdColorBlue600 } from '@zendeskgarden/css-variables';
+
+export const StyledItemWrapper = styled.div`
+  margin: 2px;
+`;
+
+export const StyledMultiselectInput = styled(StyledInput)<{
+  isSmall: boolean;
+  isVisible: boolean;
+}>`
+  && {
+    flex-basis: 60px;
+    flex-grow: 1;
+    margin: 2px;
+    width: inherit;
+    min-width: 60px;
+    line-height: ${props => (props.isSmall ? 20 / 14 : 32 / 14)};
+  }
+
+  ${props =>
+    !props.isVisible &&
+    `
+  &&& {
```

Comment:
Ooof how come we need to triple class here?

Reply:
There is very high specificity with the `css-forms` usage. These overrides are already in use in `react-autocomplete`